MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Muon/ALCHelper.h

```diff
@@ -18,6 +18,8 @@ namespace ALCHelper
   /// Create Qwt curve data from a workspace
   boost::shared_ptr<QwtData> curveDataFromWs(MatrixWorkspace_const_sptr ws, size_t wsIndex);
 
+  std::vector<boost::shared_ptr<QwtData>> curveDataFromWs(MatrixWorkspace_const_sptr ws);
+
```

Comment:
This is a modification to the ALCHelper that needs a comment. It could affect this other interface.